docs/ECLWatch/ECLWa_mods/ECLWatchSrc.xml

```diff
@@ -480,6 +480,16 @@
               query. If a Roxie query references foreign files and this is not
               enabled, publish will fail.</para>
             </listitem>
+
+            <listitem>
+              <para><emphasis role="bold">Update Super Files:</emphasis>
+              Affects the publish action behavior when a query using
+              superfiles is published with a remote dali ip or the superfile
+              is foreign. If it is not checked a referenced superfile that
+              doesn't already exist locally is created and copied from the
+              remote dali. If it is checked then the superfile is overwritten
```

Comment:
Hmmmm.  I'm not sure about the wording because is sounds a bit either or.  That is:

1. If it is not checked a referenced superfile that doesn't already exist locally is created and copied from the remote dali. 
2. If it is checked then the superfile is overwritten

#1 always happens no matter what.
#2 only happens if checked.

So something like:

> Affects the publish action behavior.  When a query using superfiles is published with a remote dali ip or the superfile is foreign, super files will be copied from the remote dali.

> If the superfiles already exist locally then if this field is checked the current definition will be overwritten.  If it is not checked the current definition will not be changed.

But please improve the wording I didn't spend much time thinking through it.